### PR TITLE
Ref 5179 create ui

### PR DESCRIPTION
**Pull request by tungct97**
#2 Ref_5179_create Ui
https://edu-redmine.framgia.vn/issues/5179
device-2019-01-09-145614

### CODE REVIEW

**Comment by tungct97** [comment]
ready

File: app/src/main/java/com/framgia/music_51/screen/ListMusicFragment.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        mBinding = DataBindingUtil.inflate(
+                inflater, R.layout.fragment_list_music, container, false);
+        View view = mBinding.getRoot();
+        return view;
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/music_51/screen/ListMusicFragment.java]: return binding.getRoot

File: app/src/main/java/com/framgia/music_51/screen/PersonFragment.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        mBinding = DataBindingUtil.inflate(
+                inflater, R.layout.fragment_person, container, false);
+        View view = mBinding.getRoot();
+        return view;
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/music_51/screen/PersonFragment.java]:
```suggestion
        return mBinding.getRoot();;
```

File: app/src/main/java/com/framgia/music_51/screen/home/HomeFragment.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        mBinding = DataBindingUtil.inflate(
+                inflater, R.layout.fragment_home, container, false);
+        View view = mBinding.getRoot();
+        return view;
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/music_51/screen/home/HomeFragment.java]:
```suggestion
        return mBinding.getRoot();
```

**Pull request merged**